sklearn/multioutput.py

```diff
@@ -0,0 +1,151 @@
+"""
+This module implements multioutput regression.
```

Comment:
This started live in `multiclass.py` but then suggested moving to `multioutput.py` in light of the work in #6127. I have no strong opinion either way.